sympy/physics/control/lti.py

```diff
@@ -1656,31 +1656,61 @@ def __neg__(self):
         return MIMOParallel(*arg_list)
 
 
-class Feedback(Basic):
-    """
-    A class for representing negative feedback interconnection between two
-    input/output systems. The first argument, ``num``, is called as the
-    primary plant or the numerator, and the second argument, ``den``, is
-    called as the feedback plant (which is often a feedback controller) or
-    the denominator. Both ``num`` and ``den`` can either be ``Series`` or
+class Feedback(SISOLinearTimeInvariant):
+    r"""
+    A class for representing closed-loop feedback interconnection between two
+    SISO input/output systems. The first argument, ``plant``, is the
+    primary plant of the closed-loop system or in simple words, the transfer
+    function representing the system/process to be controlled. Most of the time, a controller system is also
+    placed in series with the plant. This controller is known as the **plant controller** as it controls
```

Comment:
Do you need all this explanation here? Feels a bit more like a textbook rather than a docstring.

Reply:
I can add this explanation in an explanation section as I did in `TransferFunction`'s doc.
@namannimmo10?